pynucastro/rates/rate.py

```diff
@@ -505,12 +512,57 @@ def filter(self, filter_spec):
 
 
 class RateFilter(object):
-    """RateFilter stores selection rules specifying a rate or group of
-    rates to assist searching for rates stored in a Library."""
+    """RateFilter filters out a specified rate or set of rates
+    
+    A RateFilter stores selection rules specifying a rate or group of
+    rates to assist in searching for rates stored in a Library.
+    """
 
     def __init__(self, reactants=None, products=None, exact=True,
                  reverse=None, min_reactants=None, max_reactants=None,
                  min_products=None, max_products=None):
+        """Create a new RateFilter with the given selection rules
+
+        Keyword Arguments:
+            reactants -- Description of the reactants as one of:
+                1) a list of Nucleus objects
+                2) a list of string descriptions of reactant nuclides
```

Comment:
this will be rendered by Sphinx, I think the ReST style for numbered lists is `1.`, not `1)`